cmd/frostfs-adm/internal/modules/morph/root.go

```diff
@@ -323,4 +323,6 @@ func init() {
 	depositNotaryCmd.Flags().String(walletAccountFlag, "", "Wallet account address")
 	depositNotaryCmd.Flags().String(refillGasAmountFlag, "", "Amount of GAS to deposit")
 	depositNotaryCmd.Flags().String(notaryDepositTillFlag, "", "Notary deposit duration in blocks")
+
+	RootCmd.AddCommand(netmapCandidatesCmd)
```

Comment:
why have you decided to init flags in a separate `init` and not like it is already done here?

Reply:
The idea was to reduce amount of lines here, also we  are using the same approach in `cli`.